### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ggraffieti**
Hey, look at my wonderful implementation of Bar class! 
I think this improves the usability of your project very, very much :)
Regards 
Gabriele

### CODE REVIEW

File: src/main/java/Bar.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ *  Created by gab on 16/05/17.
```
Comment (maffone) [inline src/main/java/Bar.java]: A better JavaDoc is needed

File: src/main/java/Bar.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+ */
+public class Bar {
+    public static void main(String[] args) {
+        System.out.println("Bar");
```
Comment (maffone) [inline src/main/java/Bar.java]: Use a logger instead of a direct call to System.out

File: src/main/java/Bar.java (hunk 1 above)
Comment (ggraffieti) [reply src/main/java/Bar.java]: fixed

File: src/main/java/Bar.java (hunk 2 above)
Comment (ggraffieti) [reply src/main/java/Bar.java]: fixed

**Comment by maffone** [comment]
Good job

**Comment by ggraffieti** [comment]
Thank you :)

**Pull request merged**